analyses/sample-distribution-analysis/01-tumor-descriptor-and-assay-count-plot.Rmd

```diff
@@ -0,0 +1,534 @@
+    mutate(cancer_group =gsub("_..*", "", cancer_group)) %>% 
+    mutate(tumor_descriptor = match_id) %>% 
+    mutate(tumor_descriptor =gsub("..*_", "", tumor_descriptor)) %>% 
+    mutate(tumor_descriptor = case_when(grepl("Primary Tumor", tumor_descriptor) ~ "Diagnosis",
+                                 grepl("Initial CNS Tumor", tumor_descriptor) ~ "Diagnosis",
+                                 grepl('Progressive Disease Post-Mortem', tumor_descriptor) ~ 'Deceased', 
+                                 TRUE ~ tumor_descriptor)) %>%
+  arrange(Kids_First_Participant_ID)
+
+
+tumor_descriptor_df %>%
+  group_by(tumor_descriptor, experimental_strategy_sum, ) %>%
+  tally() %>%
+  arrange(desc(n)) %>% 
+  regulartable() %>%
+  fontsize(size = 12, part = "all")
+
+```
+
+The following table was supposed to provide matched different time points and summarize the information under the "tumor_descriptor_sum" column.
+But it doesn't work for now.
+
+Also, notice how for the first patient with PT_00G007DM, there are two different cancer_group assigned, which creates double events per assay.
+Why does this happen? Maybe to exclude cancer_group in match_id?
+
+
+```{r}
+
+tumor_descriptor_df_Diagnosis <- tumor_descriptor_df  %>%
+  mutate(tumor_descriptor_sum=case_when(
+    tumor_descriptor=="Diagnosis" & tumor_descriptor=="Progressive" ~ "Diagnosis-Progressive",
+    tumor_descriptor=="Diagnosis" & tumor_descriptor=="Recurrence" ~ "Diagnosis-Recurrence",
+    tumor_descriptor=="Diagnosis" & tumor_descriptor=="Deceased" ~ "Diagnosis-Deceased",
+    tumor_descriptor=="Diagnosis" & tumor_descriptor=="Second Malignancy" ~ "Diagnosis-Second Malignancy",
+    tumor_descriptor=="Diagnosis" & tumor_descriptor=="Not Reported" ~ "Diagnosis-Not Reported",
+    tumor_descriptor=="Diagnosis" & tumor_descriptor=="NA" ~ "Diagnosis-NA",
+    tumor_descriptor=="Diagnosis" & tumor_descriptor=="Residual" ~ "Diagnosis-Residual",
+    tumor_descriptor=="Diagnosis" & tumor_descriptor=="Metastatic" ~ "Diagnosis-Metastatic",
+    tumor_descriptor=="Diagnosis" & tumor_descriptor=="Unvailable" ~ "Diagnosis-Unvailable",
+    TRUE ~ "Unmatched"
+  ))
+
+
+# Save tumor_descriptor_df_Diagnosis
+write_tsv(tumor_descriptor_df_Diagnosis, path = output_paired_Diagnosis)
+
+
+tumor_descriptor_df_Diagnosis %>%
+  regulartable() %>%
+  fontsize(size = 12, part = "all")
+``` 
+
+!!!!!!!!!
+ATTENTION
+This is not working properly. They are matched time points within patients with paired assays.
+For example, patient with Kids_First_Participant_ID = PT_00G007DM.
+!!!!!!!!!
+
+## Genomic assays
+Setting aside the RNA-Seq for the moment and only looking at the patient samples that contain WXS and WGS assays. We’re collapsing the different values in tumor_descriptor to form a single descriptor when there are multiple types of tumors from the same individual.
+
+
+```{r}
+genomic_df <- tumor_descriptor_df %>%
+  filter(!(experimental_strategy_sum == "RNA")) %>% 
```

Comment:
```suggestion
   filter(!grepl("RNA", experimental_strategy_sum)) %>%
```

Reply:
Here, we want to exclude samples with RNA-seq assay only. If samples have both assays, then we will keep them.
The suggested code will remove the samples with RNA paired with DNA as well. Is this what you wanted to suggest here?